### ISSUE TITLE
Output CSV is not compatible with Google Spreadsheets

### ISSUE THREAD

**Issue by Slayer95** [marek-saji/po-csv#14]
### What happened?

If ``msgid`` includes starting or trailing whitespace, Google Spreadsheets will trim them on import. According to some sources, Microsoft Excel versions prior to 2003 also display this behavior. Note that, while RFC 4180 recognizes that there is no single true standard for CSV, it also recommends applying the robustness principle.

Test case, the following ``untranslated.csv``

```
msgctxt,msgid,msgid_plural,flags,references,extractedComments,comments,msgstr[0]
, text fragment ,,,test\main.txt:1,,,
```

Refs #10

**Comment by Slayer95** [comment]
:eyes:

**Comment by marek-saji** [comment]
@Slayer95 Sorry for late reaction. Now I understand the problem. :slightly_smiling_face: Fixed by adding quotes in all CSV fields. Released 1.0.4 containing the fix.

**Comment by Slayer95** [comment]
Thanks!
